modules/hivemq-edge-module-modbus/src/test/java/com/hivemq/edge/adapters/modbus/ModbusProtocolAdapterTest.java

```diff
@@ -65,4 +68,39 @@ void test_captureDataSample_expectedPayloadPresent()
         assertThatJson(payloadAsString).node("value").isString().isEqualTo("hello world");
     }
 
+    @Test
+    void test_deltaSamples() {
+
+        final ModBusData data1 = createSampleData(10);
+        final ModBusData data2 = createSampleData(10);
+
+        Assertions.assertEquals(0,
+                AdapterDataUtils.margeChangedSamples(data1.getDataPoints(), data2.getDataPoints()).size(), "There should be no deltas");
```

Comment:
Should this method really be named margeChangedSamples? I couldnt find an explanation for the word. Is this a typo and it should be named merge..?

Reply:
Typo! Doh! Will fix